app/src/main/res/navigation/navigation.xml

```diff
@@ -53,7 +53,14 @@
             android:id="@+id/graphFragment"
             android:name="io.mochahub.powermeter.graph.GraphFragment"
             android:label="graph_fragment"
-            tools:layout="@layout/graph_fragment" />
+            tools:layout="@layout/graph_fragment" >
+        <argument
+                android:name="exerciseID"
+                app:argType="string" />
+        <argument
+                android:name="personalRecord"
+                app:argType="string" />

```

Comment:
Can't have a `Double` type in `navargs`.

Reply:
We used Float in `ExerciseFragment` to `ExerciseDialog` args, would be nice if one or the other was standardized across the app

Reply:
Changed here https://github.com/power-meter/android-client/pull/72/commits/3a834c4baf511bf0bcf4ee0df05fad312be6af38